tests/test_documents.py

```diff
@@ -1,4 +0,0 @@
-def test_documents():
-    from flask_common import documents
-    # Just making sure import works
```

Comment:
I guess my fault for under-documeting this. The intention is to syntax-check it against python3. Even if the code isn't fully compatible yet, it makes sure that py3 can at least parse it without failing with SyntaxError (old `raise a,b,c` syntax for example)

Reply:
Aha, gotcha. Brought the test back in 52ad89d.